config/plugins/visualizations/ngl/package.json

```diff
@@ -1,23 +1,3 @@
 {
-    "name": "ngl_load",
-    "version": "0.1.0",
-    "description": "NGL visualization plugin.",
-    "keywords": [
-        "galaxy",
-        "visualization"
-    ],
-    "license": "AFL-3.0",
-    "dependencies": {
-        "babel-preset-env": "^1.6.1",
-        "backbone": "^1.3.3",
-        "bootstrap": "^3.3.7",
-        "jquery": "^3.1.1",
-        "ngl": "^0.10.4"
-    },
-    "scripts": {
-        "build": "parcel build src/ngl.js --dist-dir static/dist"
-    },
-    "devDependencies": {
-        "parcel": "^2.12.0"
-    }
+    "name": "galaxy-visualization-ngl"
```

Comment:
Was this being used anywhere?  I'd prefer to just dump it since there is no package here anymore.

Reply:
By default a package.json is needed in the directory where the node package is being installed, otherwise a parent directory further up the hierarchy is used.